egui/src/widgets/plot/mod.rs

```diff
@@ -294,34 +332,50 @@ impl Plot {
     /// .show(ui, |plot_ui| plot_ui.line(line));
     /// # });
     /// ```
-    pub fn custom_label_func(
+    pub fn label_formatter(
+        mut self,
+        label_formatter: impl Fn(&str, &Value) -> String + 'static,
+    ) -> Self {
+        self.label_formatter = Some(Box::new(label_formatter));
+        self
+    }
+
+    /// Show the pointer coordinates in the plot.
+    pub fn coordinates_formatter(
         mut self,
-        custom_label_func: impl Fn(&str, &Value) -> String + 'static,
+        position: Corner,
+        formatter: CoordinatesFormatter,
     ) -> Self {
-        self.custom_label_func = Some(Box::new(custom_label_func));
+        self.coordinates_formatter = Some((position, formatter));
         self
     }
 
-    /// Provide a function to customize the labels for the X axis.
+    /// Provide a function to customize the labels for the X axis based on the current value range.
```

Comment:
Is this the visible range (i.e. the bounds)?

```suggestion
    /// Provide a function to customize the labels for the X axis based on the current visible value range.
```